sklearn/tree/tree.py

```diff
@@ -272,10 +275,23 @@ def fit(self, X, y, sample_weight=None, check_input=True,
             min_weight_leaf = (self.min_weight_fraction_leaf *
                                np.sum(sample_weight))
 
-        if self.min_impurity_split < 0.:
+        if self.min_impurity_split is not None:
```

Comment:
Is there a deprication decorator which can be used? I know there is one for depricated functions, but I'm not sure about parameters.